packages/apps/storybook/.babelrc.json

```diff
@@ -0,0 +1,62 @@
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "shippedProposals": true,
+        "loose": true
+      }
+    ],
+    "@babel/preset-typescript",
+    "@babel/preset-react"
+  ],
+  "plugins": [
```

Comment:
With webpack 4 and babel v6, storybook was providing its own babel configurations but with webpack 5 and babel v7 we need to provide it ourselves. 

Yeah vite is easier and I recommend we use vite with storybook 7 instead of webpack.